1-exercises/B-setting-values/exercise2.js

```diff
@@ -27,6 +23,13 @@ let student = {
 
 // write code here
 
+let student = {
```

Comment:
Please use bracket notation here - the exercise needs to be done dynamically, not hard coded.